tsconfig.json

```diff
@@ -1,4 +1,12 @@
 {
   "extends": "@tsconfig/recommended/tsconfig.json",
-  "include": ["index.ts", "noble_ecc.ts"]
+  "compilerOptions": {
+    "target": "ES2016",
+    "outDir": "./dist",
```

Comment:
do we commit `dist` to git?

Reply:
we don't, but it will be published in npm package